google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerCloudMonitoringExporter.java

```diff
@@ -138,6 +138,14 @@ private CompletableResultCode exportSpannerClientMetrics(Collection<MetricData>
       return CompletableResultCode.ofFailure();
     }
 
+    // Verifies if metrics data has missing instance id.
+    if (!spannerMetricData.stream()
```

Comment:
You have to remove this negation if you are using `anyMatch`

Reply:
Why do we want to use anyMatch ? noneMatch works fine with empty streams

Reply:
Yes. both works fine. I think code should be easily readable. We have to avoid using negation in if statement if possible.

For example, `if(a < 3)` is easily readable than `if(!(a >= 3))`